unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/SearchInfo/SearchInfo.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using UnityEngine;
+
+internal interface ISearchInfo : ISearchable, ISortable<ISearchInfo>
+{
+    string id { get; }
+    string name { get; }
+    int size { get; }
+    bool isOwner { get; }
+    bool isOperator { get; }
+    bool isContributor { get; }
+    void SetId(string id);
+    void SetName(string name);
+    void SetSize(int size);
+    void SetCoords(string coords);
+    void SetRole(bool isOwner);
+}
+
+internal class SearchInfo : ISearchInfo
+{
+    string ISearchInfo.id => id;
+    string ISearchInfo.name => name;
+    int ISearchInfo.size => size;
+    bool ISearchInfo.isOwner => isOwner;
+    bool ISearchInfo.isOperator => isOperator;
+    bool ISearchInfo.isContributor => isContributor;
+    string[] ISearchable.keywords => keywords;
+
+    private bool isOwner;
+    private bool isOperator;
+    private bool isContributor;
+    private string id;
+    private string name;
+    private int size;
+    private string[] keywords;
+
+    public SearchInfo() { keywords = new string[2]; }
+
+    void ISearchInfo.SetId(string id) { this.id = id; }
+
+    void ISearchInfo.SetName(string name)
+    {
+        this.name = name;
+        keywords[0] = name;
```

Comment:
This is a little strange, any reason for array usage instead of a list?

Reply:
I choose to use an array cause it was a fixed size, I didn't have to heavily manipulate data stored there and because it wasn't clear at the moment if I should use an enum or constant to name the indices like `keywords[NAME] = name` which would be super strange to do using a list :P